src/pages/patientView/clinicalInformation/lib/clinicalAttributesUtil.js

```diff
@@ -207,21 +208,28 @@ function getSpanElementsFromCleanData(
     clinicalAttributesCleanDerived,
     cancerStudyId
 ) {
-    let spans = [];
-    return Object.keys(clinicalAttributesCleanDerived).map(key => {
-        let value = clinicalAttributesCleanDerived[key];
-        return (
-            <span
-                is
-                class="clinical-attribute"
-                attr-id={key}
-                attr-value={value}
-                study={cancerStudyId}
-            >
-                {value}
-            </span>
-        );
-    });
+    return Object.keys(clinicalAttributesCleanDerived)
+        .sort(styleConsts.compare)
+        .map(key => {
+            let value = clinicalAttributesCleanDerived[key];
+            return (
+                <span
+                    is
+                    attr-id={key}
+                    class="clinical-attribute"
+                    attr-value={value}
+                    style={{
+                        color: styleConsts.colorGetter(
+                            value,
+                            key,
+                            styleConsts.config
+                        ),
+                    }}
+                >
+                    {styleConsts.stringBuilder(value, key, styleConsts.config)}
```

Comment:
you can handle the comma out here where it will be possible to tell if it's last element.   one more potential issue, you don't want color to be applied to comma. maybe you can put comma in its own span and apply the black color.